ManageWikiExtensions.php

```diff
@@ -1747,6 +1747,14 @@
 		'requires' => [],
 		'section' => 'skins',
 	],
+	'bluesky' => [
+		'name' => 'BlueSky',
+		'linkPage' => 'https://www.mediawiki.org/wiki/Skin:BlueSky',
```

Comment:
Special:MyLanguage

Reply:
That is not necessary as that page has not been marked for translation.

Reply:
I'd rather we got it marked for translation

Reply:
Still it's preferable to do that so it wouldn't need to be changed later if it were.

Reply:
Anyone volunteering?

Reply:
Done.

Reply:
Thank you!